resources/providers.json

```diff
@@ -1278,5 +1278,52 @@
             "sentmail": "Sent Items",
             "trash": "Deleted Items"
         }
+    },
+    "wp.pl": {
+        "servers": {
+            "imap": [
+                     {
+                     "port": 993,
+                     "hostname": "imap.wp.pl",
+                     "ssl": true
+                     }
+                     ],
+            "smtp": [
+                     {
+                     "port": 465,
+                     "hostname": "smtp.wp.pl",
+                     "ssl": true
+                     }
+                     ]
+        },
+        "domain-match": [
+                         "wp\\.pl"
+                         ],
+        "mailboxes": {
+            "spam": "SPAM",
+            "sentmail": "Wys&AUI-ane",
+            "trash": "Kosz"
+        }
+    },
+    "o2.pl": {
+        "servers": {
+            "imap": [
+                     {
+                     "port": 993,
+                     "hostname": "poczta.o2.pl",
+                     "ssl": true
+                     }
+                     ],
+            "smtp": [
+                     {
+                     "port": 465,
+                     "hostname": "poczta.o2.pl",
+                     "ssl": true
+                     }
+                     ]
+        },
+        "domain-match": [
+                         "o2\\.pl"
+                         ]
```

Comment:
Could add the mailboxes here anyway?
For consistency, I think there's the fallback for other one too.